package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Where does this come from?

Reply:
This file was generated automatically. Didn't add it.

Reply:
We don't even use npm in this project 😓 . So please remove it